### PR TITLE
TAMAYA-351 Do not append newline (\n) at the end of a value when resolved with FileResolver

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

**Comment by ottlinger** [comment]
Thanks for your contributions - did you already file a CLA with ApacheSoftwareFoundation to properly license your commits? Thanks.

File: modules/resolver/src/main/java/org/apache/tamaya/resolver/internal/FileResolver.java [hunk 1]
```diff
@@ -86,7 +86,12 @@ public String evaluate(String expression) {
                 builder.append(inputLine).append("\n");
             }
 
-            return builder.toString();
+            String content = builder.toString();
+            if (content.endsWith("\n")) {
+                content = content.substring(0, conent.length() - 2);
```
Comment (ottlinger) [inline modules/resolver/src/main/java/org/apache/tamaya/resolver/internal/FileResolver.java]: conent? Seems like a typo to me.

Is it possible to provide/augment a test case to verify the behaviour and prevent regressions in the future. Thanks.

**Comment by alexsuter** [comment]
Fixed + Test added

**Comment by alexsuter** [comment]
Which Java Level do we have to support? Since Java 7 there are a lot of shortcut methods, to read a file: https://stackoverflow.com/questions/326390/how-do-i-create-a-java-string-from-the-contents-of-a-file

**Comment by alexsuter** [comment]
@peculater 
I use now java 7 features to load the hole file. so we won't have to handle the new line anymore.

**Comment by peculater** [comment]
Excellent.  I think that cleans up the code nicely.  Drop a license header in that Testresource3.txt file to get the tests passing and this should be good to merge. :+1:

**Comment by alexsuter** [comment]
Hmm. Really not nice for this test case... 
Try to fix that this evening

**Comment by atsticks** [comment]
I close this request, it has been fixed in the code already. Thanks.

**Pull request closed without merging**